sympy/core/tests/test_expr.py

```diff
@@ -1011,7 +1011,6 @@ def test_extractions():
     assert (Rational(1, 2)*x).extract_multiplicatively(3) == x/6
     assert (sqrt(x)).extract_multiplicatively(x) is None
     assert (sqrt(x)).extract_multiplicatively(1/x) is None
-    assert x.extract_multiplicatively(-x) is None
```

Comment:
It is generally frowned upon to delete a test; if this no longer passes it represents a regression in terms of what the routine should be doing.